package lab.user_service.security.exceptions;

public class UserAlreadyExistsException extends RuntimeException {
}
